lib/SSRDataStore.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import PropTypes from 'prop-types';
+import { EJSON } from 'meteor/ejson';
+import { generateQueryId } from './utils.js';
+export const SSRDataStoreContext = React.createContext(null);
```

Comment:
This release is going to break React 15.x, we need to make it optional. We still want to allow latest versions of grapher-react to be used with 15.x